<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queryRep</class>
 <widget class="QDialog" name="queryRep">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поиск постановок</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>НАЗВАНИЕ СПЕКТАКЛЯ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSearch">
       <property name="text">
        <string>ИСКАТЬ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text">
        <string>ЗАКРЫТЬ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>buttonSearch</tabstop>
  <tabstop>buttonClose</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>queryRep</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSearch</sender>
   <signal>clicked()</signal>
   <receiver>queryRep</receiver>
   <slot>SearchClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SearchClicked()</slot>
 </slots>
</ui>
